### PR TITLE
added missing modules in agent menu

### LINKED ISSUE #3227: Vulnerabilities module is missing in the agent menu
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 4.x   | 7.x     | 400x |

**Description**
The `Vulnerabilities` module is missing in the agent menu. As result, when the window size is small, the module can't be accessed of agent overview (when the window size is bigger, the `Vulnerabilities` module is pinned near to `More` dropdown button.

Edit: some modules as `Docker` and another in other section could are missing. Review the comments.

**Steps to reproduce**

1. Reduce the window size.
2. Go to Agents and select an agent.
3. The `Vulnerabilities` module could not be pinned (due to window size) but it is missing when open the module menu using the `Module` or `More...` dropdown buttons.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by eze9252**
Hi team, this resolve:

- Fix modules are missing in the agent menu

Captura de pantalla de 2021-05-12 16-17-59

Closes #3227

### CODE REVIEW

**Review by Machi3mfl — APPROVED** [review]
Fixed 

Screen Shot 2021-05-12 at 17 35 58

**Comment by matiasmoreno876** [comment]
LGTM
image

**Review by matiasmoreno876 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fixed screen flickers in Cluster visualization [#3159](https://github.com/wazuh/wazuh-kibana-app/pull/3159)
 - Fix the broken links when using `server.basePath` Kibana setting [#3161](https://github.com/wazuh/wazuh-kibana-app/pull/3161)
 - Fixing filter in reports [#3173](https://github.com/wazuh/wazuh-kibana-app/pull/3173)
+- Fix modules are missing in the agent menu [#3227] (https://github.com/wazuh/wazuh-kibana-app/issues/3227)
```
Comment (gabiwassan) [inline CHANGELOG.md:24]: This URL should be the one of the PR?

Comment (eze9252) [reply CHANGELOG.md:24]: Yes! I fix it now

**Review by gabiwassan — APPROVED** [review]
LGTM!

**Pull request merged**